backend/src/models/user.ts

```diff
@@ -2,8 +2,9 @@ export type User = {
   user_id: number;
   email: string;
   first_name: string;
+  last_name: string;
```

Comment:
Figured I'd just add this here since I accidentally included that in the db schema.l